toolset/kover-cli/README.md

```diff
@@ -0,0 +1,148 @@
+| --include <class-name>                | instrument only specified classes, wildcards `*` and `?` are acceptable                                                    |          |    +     |
+
+### Generating reports
+Allows you to generate HTML and XML reports from the existing binary report.
+
+`java -jar kover-cli.jar report [<binary-report-path> ...] --classfiles <class-file-path> [--exclude <class-name>] [--excludeAnnotation <annotation-name>] [--html <html-dir>] [--include <class-name>] --src <sources-path> [--title <html-title>] [--xml <xml-file-path>]`
+
+| Option                                | Description                                                                                             | Required | Multiple |
+|---------------------------------------|---------------------------------------------------------------------------------------------------------|:--------:|:--------:|
+| `<binary-report-path>`                | list of binary reports files                                                                            |          |    +     |
+| --classfiles <class-file-path>        | location of the compiled class-files root (must be original and not instrumented)                       |    +     |    +     |
+| --exclude <class-name>                | filter to exclude classes, wildcards `*` and `?` are acceptable                                         |          |    +     |
+| --excludeAnnotation <annotation-name> | filter to exclude classes and functions marked by this annotation, wildcards `*` and `?` are acceptable |          |    +     |
+| --html <html-dir>                     | generate a HTML report in the specified path                                                            |          |          |
+| --include <class-name>                | filter to include classes, wildcards `*` and `?` are acceptable                                         |          |    +     |
+| --src <sources-path>                  | location of the source files root                                                                       |    +     |    +     |
+| --title <html-title>                  | title in the HTML report                                                                                |          |          |
+| --xml <xml-file-path>                 | generate a XML report in the specified path                                                             |          |          |
+
+## Off-line instrumentation
+
+Off-line instrumentation is suitable when using runtime environments that do not support Java agents.
+It instruments the files located in the file system and saves the result to the specified directory.
+
+To run classes instrumented offline, you need to add `org.jetbrains.kotlinx:kover-offline` artifact to the application's classpath.
+
+You also need to pass the system property `kover.offline.report.path` to the application with the path to indicate the path where you want binary reports to be saved.
```

Comment:
`...to the application with the path to indicate the path...`
First 'path' seems excessive :)